src/pages/notFound/ui/NotFoundPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {NavLink} from "react-router-dom"
+import cl from "./NotFoundPage.module.scss"
+import { getFeedback, getMain } from 'app/providers/router/routeConfig/routes';
+import { Button } from 'shared/ui/button/Button';
+
+const NotFoundPage = () => {
+    return (
+        <main className={cl.NotFoundPage}>
+            <span className={cl.mainText}>Ой! Такой страницы, к сожалению, не существует ((</span>
```

Comment:
кандидат на h1, но никак не span

```suggestion
            <h1 className={cl.mainText}>Ой! Такой страницы, к сожалению, не существует ((</h1>
```